cmd/main.go

```diff
@@ -132,6 +142,19 @@ func parseCliArgs(flags *args) (*cobra.Command, error) {
 		return nil, err
 	}
 
+	cobra.OnInitialize(func() {
+		conf, err = loadConfig(cfgFile)
+		if err != nil {
+			fmt.Printf("error reading config: %v\n", err)
```

Comment:
This should be logged using the klog.Info logger (after the verbose flag was initialized). Its only informal and should not be printed each time if there is no config.